GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -1832,7 +1832,7 @@ private void UserManualToolStripMenuItemClick(object sender, EventArgs e)
             try
```

Comment:
Other try-catch removed, this kept?

Reply:
Good catch. This one slipped through. I have removed it.